tck/src/main/java/org/eclipse/microprofile/openapi/apps/petstore/resource/UserResource.java

```diff
@@ -110,9 +109,8 @@ public Response createUsersWithListInput(
     })
     public Response updateUser(
             @Parameter(name = "username", description = "name that need to be deleted",
-                       schema = @Schema(type = SchemaType.STRING),
-                       required = true) @PathParam("username") String username,
```

Comment:
Should we keep the `required = true` for the `@Parameter`? I know the default is `true` for path parameters anyway, but technically it's a separate change from the linked issue.

Reply:
Yes, I think that's a mistake.

Reply:
I've made this change and rebased on `main` to resolve conflicts.